.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+language: node_js
+node_js:
+  - '10'
+  - '12'
+# keep the npm cache to speed up installs
+cache:
```

Comment:
Does `cache: yarn` not work here?

Reply:
copied from another project, i can change